src/west/app/project.py

```diff
@@ -153,6 +153,16 @@ def __init__(self):
 branch will be used, if it exists.
 
 With neither, -m {MANIFEST_URL_DEFAULT} is assumed.
+
+Warning: 'west init' renames and/or deletes temporary files inside the
+workspace being created. This fails on some filesystems when some
+development tool or any other program is trying to read/index these
+temporary files at the same time. For instance, it is required to stop
+Visual Studio Code before running 'west init` on the Windows NTFS
+filesystem. Find other, similar "Access is denied" examples in west
+issue #558.
```

Comment:
```suggestion
filesystem. Find other, similar "Access is denied" examples in 
https://github.com/zephyrproject-rtos/west/issues/558.
```

Most terminals make links clickable these days, so this is worth spelling out, IMO

Reply:
I considered this but I tried to minimize the length as much as possible. In fact I was a bit reluctant to add this _at all_ because a help text should be really concise and not turn into a changelog or bug list. However this particular bug seems frequent, fatal and totally unfixable.